src/web/src/components/Posts/Post.tsx

```diff
@@ -98,6 +101,25 @@ const useStyles = makeStyles((theme: Theme) =>
         marginLeft: '.1em',
       },
     },
+    postCount: {
+      marginRight: '16px',
+      display: 'flex',
+      justifyContent: 'flex-end',
+      [theme.breakpoints.down(1024)]: {
+        margin: '0 2em 0.05em 0',
+      },
+      [theme.breakpoints.down(600)]: {
+        margin: '0 0 0.05em 0',
+      },
+    },
+    chipComponent: {
+      border: `1px solid ${theme.palette.border.dark}`,
```

Comment:
I think it looks better to use primary and secondary colors here.

```suggestion
import { useTheme} from '../ThemeProvider';
 ...
const { themeName } = useTheme();
...
<Chip
    ....
    color={themeName === 'light'  ? "primary": "secondary"}
/>
```
By doing this, you can remove the added color to our palette.

Reply:
I tried it before but the color isn't what I'm looking for.

Reply:
I personally think these predefined colors look great. It's also consistent with elements besides it.
<img width="1000" alt="Screen Shot 2021-11-30 at 4 21 51 PM" src="https://user-images.githubusercontent.com/51073515/144130150-4407f16e-4e51-4581-bc65-34a42089e787.png">
<img width="1000" alt="Screen Shot 2021-11-30 at 4 21 45 PM" src="https://user-images.githubusercontent.com/51073515/144130151-c491ec94-dfc6-4294-8494-2a74dcfe9a08.png">

Reply:
If @humphd is okay, I can change it to those colors

Reply:
Yeah, sounds good